### ISSUE TITLE
[Feature Request] Support pnpm

### ISSUE THREAD

**Issue by Aqours** [bmish/check-dependency-version-consistency#573]
See [pnpm workspaces](https://pnpm.io/workspaces)

**Comment by bmish** [comment]
It's a a good idea to support pnpm. I briefly started working on this in #574.

**Comment by kachkaev** [comment]
I’d love to use ` check-dependency-version-consistency` in a pnpm monorepo!  https://github.com/bmish/check-dependency-version-consistency/pull/574 looks like a great start and it should cover simple scenarios. How about shipping a minimal version first and then covering more nuanced scenarios (`file:`, `link:`, etc.)?

**Comment by bmish** [comment]
Good point. I'll try to finish up this minimal version in the next week.

**Comment by bmish** [comment]
Implemented in https://github.com/bmish/check-dependency-version-consistency/releases/tag/v3.1.0.

Feel free to comment here to let us know if it works properly.

**Comment by kachkaev** [comment]
Amazing, thanks a lot @bmish! I tried v3.1.0 and it worked like a charm. Here is the first output of
`pnpm check-dependency-version-consistency` in our pnpm monorepo:

```
Found 1 dependency with mismatching versions across the workspace. Fix with `--fix`.
╔════════╤════════╤════════════════════════════════════════╗
║ eslint │ Usages │ Packages                               ║
╟────────┼────────┼────────────────────────────────────────╢
║ 8.39.0 │ 2      │ @apps/frontend, @apps/proxy-website    ║
╟────────┼────────┼────────────────────────────────────────╢
║ 8.37.0 │ 2      │ @apps/advanced-menu, @apps/user-script ║
╚════════╧════════╧════════════════════════════════════════╝

 ELIFECYCLE  Command failed with exit code 1.
```

Running `pnpm check-dependency-version-consistency --fix` updated `package.json` files, just like it did when I used Yarn 🎉 I just had to run `pnpm install` to update the `pnpm-lock.json` and that was it!

Interestingly, there was a small disparity in repo requirements between lint and fix modes. When I entered `--fix` for the first time, I got this error: `` package.json missing `name` ``. Adding `"name": "my-monorepo"` to the root `package.json` helped as a workaround. This is not a blocker of any kind, just a curiosity worth sharing. If this extra requirement to the root `package.json` file is not critical, happy to test a new version for you!

**Comment by bmish** [comment]
@kachkaev good catch, the name shouldn't be required, that was a bug, I'll release the fix momentarily: #590
